docs/plugins/composable_resources.md

```diff
@@ -197,6 +197,13 @@ To modify an existing passthru resource to be written to only after other eligib
 irods@hostname:~/ $ iadmin modresc newResc context 'write=0.4;read=1.0'
 ```
 
+Setting a `read` or `write` weight to `0` will prevent that action from occurring on the branch of the resource tree under the passthru (because a `0` vote signals that it cannot service the incoming request).  The following will 'turn off' the resource `newResc` (a policy-based way of marking a resource "down"):
+
+
+```
+irods@hostname:~/ $ iadmin modresc newResc context 'write=0.0;read=0.0'
```

Comment:
Should we show what this looks like, or would that possibly change over time? Something like...
```bash
irods@hostname:~/ $ ilsresc
<tree>
irods@hostname:~/ $ iadmin modresc newResc context 'write=0.0;read=0.0'
irods@hostname:~/ $ iput -R newResc newfile
<error>
irods@hostname:~/ $ iget -R newResc oldfile_with_one_replica
<error>
```

Reply:
This seems like something that could be good for the policy cookbook.

Reply:
definitely - showing results is ... out of scope for this part of the docs i think

Reply:
Makes sense to me.